.github/workflows/release_builds.yml

```diff
@@ -88,7 +88,7 @@ jobs:
         working-directory: ./otelcolbuilder/cmd
         run: |
           go version -m ${{ steps.set_filename.outputs.filename }} | \
-          grep -E "/(receiver|exporter|processor|extension)/" | \
+          { grep -E "/(receiver|exporter|processor|extension)/" || true; } | \
           tee otelcol-sumo-${{matrix.arch_os}}_modules.txt
 
```

Comment:
@swiatekm-sumo not sure, why do we have `Show included modules` in CI?

Reply:
I think this was a way to verify we actually built with the modules we intended. But this can be done by building the binary and running `otelcol-sumo components` now as well.